src/blob_gc_job.cc

```diff
@@ -48,6 +52,12 @@ class BlobGCJob::GarbageCollectionWriteCallback : public WriteCallback {
       }
     }
 
+    if (!s.ok()) {
+      RecordTick(stats_, BLOB_DB_BLOB_INDEX_EVICTED_COUNT);
+      RecordTick(stats_, BLOB_DB_BLOB_INDEX_EVICTED_SIZE,
+                 key_.size() + index_entry.size());
```

Comment:
`BLOB_DB_GC_NUM_KEYS_OVERWRITTEN` and `BLOB_DB_GC_BYTES_OVERWRITTEN` is more suitable I think.